feature/qos/tests/qos_ecn_config_test/qos_ecn_config_test.go

```diff
@@ -256,11 +256,13 @@ func testCiscoECNConfig(t *testing.T) {
 	dut := ondatra.DUT(t, "dut")
 	d := &oc.Root{}
 	q := d.GetOrCreateQos()
-	queueName := []string{"a_NC1", "b_AF4", "c_AF3", "d_AF2", "e_AF1", "f_BE0", "g_BE1"}
+	queueName := []string{"NC1", "AF4", "AF3", "AF2", "AF1", "BE0", "BE1"}
 
-	for _, queue := range queueName {
+	for i, queue := range queueName {
 		q1 := q.GetOrCreateQueue(queue)
 		q1.Name = ygot.String(queue)
+		queueid := 7 - i
```

Comment:
7 is magic number to me.. 
If it is a fixed value, please make it a constant with a meaning name.
If it is the size of the "queueName" list, please use len().

Reply:
fixed